Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBox.cs

```diff
@@ -220,11 +245,13 @@ protected virtual void Update()
 
         protected virtual void CreateTransforms()
         {
-            if (scaleTransform == null) {
+            if (scaleTransform == null)
+            {
                 scaleTransform = transform.Find("Scale");
```

Comment:
Rather than finding the scale transform, consider using localScale or lossyScale, whichever is most appropriate

Reply:
Which "Scale" does this map to? localScale or lossyScale? Can the code directly access the appropriate one? It will help with understanding the side effect of the scale transform.